src/apiCalls.js

```diff
@@ -0,0 +1,14 @@
+export const fetchAllMovies = () => {
+  return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => response.json())
+  }
+
+export const fetchMovieDetails = (movieID) => {
+  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${movieID}`)
+    .then(response => response.json())
+}
+
+export const fetchMovieVideos = (movieID) => {
+  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${movieID}/videos`)
+    .then(response => response.json())
+}
```

Comment:
Clean!
Would still be interested if we could get some further input / feedback on bringing more of the fetch functionality back into this file, as for now we have only broken out the initial call to return those parsed promises.
Is there a way we could / should be doing more in this file before returning to where the calls are made from App.js (And Details.js, respectively)?